### DISCUSSION TITLE
multiple select chips

### DISCUSSION THREAD

**Discussion by gocs** [saadeghi/daisyui#2111] [Ideas / feature requests]
Multiple select dropdown just like [chosen](https://harvesthq.github.io/chosen/) without making my own?

Sample:
![image](https://github.com/saadeghi/daisyui/assets/27877342/5a0e083f-34f8-473d-82ce-c578ab90e7c9)
![image](https://github.com/saadeghi/daisyui/assets/27877342/1cc6beb0-0ef2-4f01-b216-6aa725a0b4a4)

If this is already done somewhere, let me know.

**Comment by saadeghi** [comment]
daisyUI can't do this because there's no native HTML element like this, and something like this needs JS and a combination of some HTML elements (buttons, divs, lists, text input, etc)   

There are JS plugins for different frameworks (or pure JS). You can find them on NPM site or GitHub.  
If they allow adding custom class names, you can add daisyUI class names to make the style fit your site.  
If they don't allow custom class names, you can just add a few CSS lines to add daisyUI color names the their default style.

Let me know if you have any questions.

**Reply by pannet1** [reply]
most of the js i tried does not allow to create multiselect in every row of a table for example. Even if they use class for applying the css, the styling happens only to the first row. i tried choices and Tom Select.
